core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/kit/KitDescriptor.java

```diff
@@ -76,7 +76,7 @@
   @Transient
   private final Collection<ChangeLog> changelog = new ArrayList<>();
 
-  @OneToOne(targetEntity = UserImpl.class)
+  @OneToMany(targetEntity = UserImpl.class)
```

Comment:
I had noted this should be OneToMany, but it should actually be ManyToOne. My bad